SectionedCollectionView/Views/CollectionViewCell.swift

```diff
@@ -0,0 +1,52 @@
+//
+//  CollectionViewCell.swift
+//  SectionedCollectionView
+//
+//  Created by Esteban Arrua on 6/8/18.
+//  Copyright © 2018 Hattrick. All rights reserved.
+//
+
+import Foundation
+import UIKit
+
+class CollectionViewCell: UICollectionViewCell {
+    
+    static let nibName = "CollectionViewCell"
+    static let cellReuseIdentifier = "CollectionViewCell"
+    
+    private let gradientStartColor = UIColor(red: 46/255, green: 132/255, blue: 250/255, alpha: 1)
+    private let gradientEndColor = UIColor(red: 83/255, green: 99/255, blue: 236/255, alpha: 1)
+    private var gradientBackgroundLayer: CAGradientLayer?
+    
+    @IBOutlet weak var nameLabel: UILabel!
+    
+    func configure(with name: String, selected: Bool) {
+        nameLabel.text = name
+        nameLabel.textColor = selected ? .white : .black
+        backgroundColor = .white
```

Comment:
This and the lines below (permanent style setups) should be done once and not every time configure is called. Maybe from awakeFromNib? Also call a method like setupStyle or something like that